__tests__/lib/ckpool.test.ts

```
@@ -0,0 +1,167 @@
+import {
+    CKPoolAPI,
+    CKPoolError,
+    CKPoolErrorCode,
+} from '../../lib/ckpool';
```

ESLint

[error] Replace `⏎····CKPoolAPI,⏎····CKPoolError,⏎····CKPoolErrorCode,⏎` with `·CKPoolAPI,·CKPoolError,·CKPoolErrorCode·`

(prettier/prettier)